src/server/rdb_load.cc

```diff
@@ -1926,14 +1926,13 @@ error_code RdbLoaderBase::HandleJournalBlob(Service* service, DbIndex dbid) {
   SET_OR_RETURN(FetchGenericString(), journal_blob);
 
   io::BytesSource bs{io::Buffer(journal_blob)};
-  JournalReader rd{dbid};
```

Comment:
what about the dbid? How is this working now?

Reply:
Indeed 😅

Reply:
This should be catchable by tests once we have the new testing data generator